src/core/pos.ml

```diff
@@ -17,34 +13,13 @@ and pos_data =
 (* NOTE laziness is essential on big files (especially those with long lines),
    because computing utf8 positions is expensive. *)
```

Comment:
If you remove lazyness, this comment should be removed as well. But are you sure you want to remove lazyness on positions?

Reply:
Lazyness would be handled by the lexer, Sedlex doesn't use it, so we don't.